testData/solutions/win32.json5

```diff
@@ -5490,249 +5490,892 @@
                     },
-                    "AppStarting": {
+                    "Hand": {
                         "transform": {
                             "type": "fluid.transforms.quantize",
                             "inputPath": "http://registry\\.gpii\\.net/common/cursorSize",
                             "ranges": [
                                 {
                                     "upperBound": 0.333,
-                                    "output": "%SystemRoot%\\cursors\\aero_working.ani"
-                                },
-                                {
+                                    "output": {
+                                        "transform": {
+                                            "type": "fluid.transforms.valueMapper",
+                                            "defaultInputPath": "http://registry\\.gpii\\.net/common/cursorColor",
+                                            "match": {
+                                                "ReverseBlack": "%SystemRoot%\\cursors\\aero_link_i.cur"
+                                            },
+                                            "noMatch": {
+                                                "outputValue": ""
+                                            }
+                                        }
+                                    }
+                                }, {
                                     "upperBound": 0.666,
-                                    "output": "%SystemRoot%\\cursors\\aero_working_l.ani"
-                                },
-                                {
-                                    "output": "%SystemRoot%\\cursors\\aero_working_xl.ani"
+                                    "output": {
+                                        "transform": {
+                                            "type": "fluid.transforms.valueMapper",
+                                            "defaultInputPath": "http://registry\\.gpii\\.net/common/cursorColor",
+                                            "match": {
+                                                "ReverseBlack": "%SystemRoot%\\cursors\\aero_link_im.cur"
+                                            },
+                                            "noMatch": {
+                                                "outputValue": ""
+                                            }
+                                        }
+                                    }
+                                }, {
+                                    "output": {
+                                        "transform": {
+                                            "type": "fluid.transforms.valueMapper",
+                                            "defaultInputPath": "http://registry\\.gpii\\.net/common/cursorColor",
+                                            "match": {
+                                                "ReverseBlack": "C:\\Windows\\cursors\\aero_link_il.cur"
```

Comment:
Presumably this should refer to %SystemRoot% like the others

Reply:
Really nice catch